cpp/src/Ice/OutgoingAsync.cpp

```diff
@@ -418,6 +418,8 @@ ProxyOutgoingAsyncBase::cancelable(const CancellationHandlerPtr& handler)
 void
 ProxyOutgoingAsyncBase::retryException()
 {
+    // retryException is only called by ConnectRequestHandler!
```

Comment:
It's not easy to do as we need to get hold of the RequestHandlerCache and ProxyOutgoingAsyncBase doesn't provide access to its proxy. 

So this would be done in a broader refactoring of Outgoing and friends.